app/src/test/java/org/simple/clinic/summary/teleconsultation/contactdoctor/ContactDoctorUpdateTest.kt

```diff
@@ -44,15 +48,88 @@ class ContactDoctorUpdateTest {
         .given(model)
         .whenEvent(TeleconsultRequestCreated(
             teleconsultRecordId = teleconsultRecordId,
-            doctorPhoneNumber = doctorPhoneNumber
+            doctorPhoneNumber = doctorPhoneNumber,
+            messageTarget = MessageTarget.SMS
         ))
         .then(assertThatNext(
             hasNoModel(),
             hasEffects(LoadPatientTeleconsultInfo(
                 patientUuid = patientUuid,
                 teleconsultRecordId = teleconsultRecordId,
-                doctorPhoneNumber = doctorPhoneNumber
+                doctorPhoneNumber = doctorPhoneNumber,
+                messageTarget = MessageTarget.SMS
             ))
         ))
   }
+
+  @Test
+  fun `when patient teleconsult info is loaded, then send teleconsult message`() {
+    val teleconsultRecordId = UUID.fromString("144a31ab-ee5f-4f06-b3e3-3ef26e0a0330")
+    val facilityUuid = UUID.fromString("ffae95ae-ae05-4bdc-8103-3ee8e927713c")
+    val facility = TestData.facility(uuid = facilityUuid)
+    val user = TestData.loggedInUser(uuid = UUID.fromString("a072f41a-9990-400f-b293-b282a6f4b8ff"))
+    val doctorPhoneNumber = "+911111111111"
```

Comment:
Here? we've added `doctorPhoneNumber` as global properties now but haven't removed them from tests?